tests/playwright/README.md

```diff
@@ -0,0 +1,9 @@
+# playwright tests
+
+## Getting started
+
+- Run `npm ci`
```

Comment:
Update README and add optional environment variables that can be set along with their defaults